vendor/engines/bulk_email/app/models/bulk_email/user_extension.rb

```diff
@@ -0,0 +1,17 @@
+module BulkEmail
+
+  module UserExtension
+
+    extend ActiveSupport::Concern
+
+    included do
+      has_many :bulk_email_jobs,
```

Comment:
Same here. Is this relation necessary and worth the overhead? I mean overhead in terms of API, not performance. And it looks like we might not even be extending this onto the User yet.